### PR TITLE
Rdm 9052

### PR DESCRIPTION
**Pull request by henrykcgi**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-9838

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/resources/db/changelog/db.changelog-RDM-9052.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
```
Comment (danlysiak) [inline src/main/resources/db/changelog/db.changelog-RDM-9052.xml:1]: What is the purpose of this changelog please?

File: src/main/resources/db/changelog/db.changelog-master.xml [hunk 2]
```diff
@@ -39,4 +39,5 @@
     <include file="db/changelog/db.changelog-RDM-8463.xml"/>
     <include file="db/changelog/db.changelog-RDM-9287.xml"/>
     <include file="db/changelog/db.changelog-RDM-9505.xml"/>
+   <include file="db/changelog/db.changelog-RDM-9052.xml"/>
```
Comment (danlysiak) [inline src/main/resources/db/changelog/db.changelog-master.xml:42]: Make indentation consistent

File: build.gradle [hunk 3]
```diff
@@ -91,12 +91,12 @@ repositories {
 ext {
     junitJupiterVersion = '5.5.2'
     junitVintageVersion = '5.5.2'
-    powermockVersion = '2.0.0-beta.5'
+    powermockVersion = '2.0.7'
     reformLogging = '5.1.0'
     appInsightsVersion = '2.4.1'
-    swagger2Version = '2.10.0'
+    swagger2Version = '3.0.0'
```
Comment (danlysiak) [inline build.gradle:97]: Fairly sure Swagger 3 has breaking changes that would require config updates. Can you please check whether Swagger is still working?

**Review by danlysiak — COMMENTED** [review]
Couple of comments/queries.

File: src/main/java/uk/gov/hmcts/ccd/SecurityConfiguration.java [hunk 4]
```diff
@@ -47,15 +47,16 @@
...
         "/health/liveness",
-        "/status/health",
+        "/health/readiness",
+        "/health",
         "/loggers/**",
-        "/"
+        "/",
+        "/swagger-resources/**",
+        "/swagger-ui/**",
+        "/swagger-ui/index.html**",
```
Comment (danlysiak) [inline src/main/java/uk/gov/hmcts/ccd/SecurityConfiguration.java:58]: Should be able to remove the second of these, as the first covers it?

**Review by danlysiak — APPROVED** [review]
Minor comment/question; approved 👍

**Review by noronhaa — APPROVED** [review]

**Pull request merged**